src/MongoDB/Server.c

```diff
@@ -362,7 +362,7 @@ static PHP_METHOD(MongoDB_Driver_Server, getLatency)
 	if (mongoc_server_description_round_trip_time(sd) == -1) {
 		RETVAL_NULL();
 	} else {
-		RETVAL_LONG((zend_long) mongoc_server_description_round_trip_time(sd));
```

Comment:
@levon80999 per your question in [PHPC-1928](https://jira.mongodb.org/browse/PHPC-1928?focusedCommentId=4804154&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4804154):

> I also saw that there are some lines in /MongoDB/ServerDescription.c file which used (zend_long) casting in lines (142 and 256). Should I remove it too ?
>
> ```
> line 142: RETVAL_LONG((zend_long) mongoc_server_description_round_trip_time(intern->server_description));
>
> line 246: ZVAL_LONG(&round_trip_time, (zend_long) mongoc_server_description_round_trip_time(intern->server_description));
> ```

Please address those instances as well in this PR.

Reply:
done